package.json

```diff
@@ -19,6 +19,12 @@
   "bugs": {
     "url": "https://github.com/katowulf/mockfirebase/issues"
   },
+  "files": [
+    "*.md",
+    "src/",
+    "tutorials/",
+    "browser/"
```

Comment:
Made especially painful by Bowers lack of an equivalent for `tar -tf $(npm pack)`:

```sh
 $ tar -tf $(npm pack) 
package/package.json
package/README.md
package/LICENSE
package/index.js
package/changelog.txt
package/dist/angularfire.js
package/dist/angularfire.min.js
```